### DISCUSSION TITLE
Prefix Database Tables

### DISCUSSION THREAD

**Discussion by jlarmstrongiv** [prisma/prisma#2232] [Q&A] [answered]
Is it possible to prefix all database tables in prisma?  Unfortunately, I am working on brownfield projects that share a database.

**Comment by ryands17** [comment]
Hey @jlarmstrongiv :wave: 
Currently Prisma doesn't support prefixing database tables, so as a workaround, you would have to prefix them in the `schema.prisma` yourself for now.

Could you pls create a Feature Request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=)

**Reply by jlarmstrongiv** [reply, marked as answer]
Sure thing!  Created https://github.com/prisma/prisma/issues/2278

**Reply by MarcusCemes** [reply]
I'm exploring Prisma as a potential ORM for our university's backend system, we don't have the luxury of a fresh DB.

I'm not too familiar with Prisma yet, by manually prefixing, do you mean setting a custom table name with `@@map`?

**Reply by jlarmstrongiv** [reply]
@MarcusCemes Back when this issue was created, `@@map` wasn’t around.  Certainly, a workaround would be to prefix all the columns by renaming all of them with `@@map`.  In a perfect world, the prefix would be to set it once, and then no need to worry about it—but that doesn’t exist.  

As for exploring prisma as a potential ORM, just some thoughts I have is that it’s been under heavy development lately.  Not everything you will need is there, but things are getting better.  The con is that the prisma 1 => prisma 2 migration really didn’t exist—I personally felt burned by that.  I’m hesitant to use it on projects that matter until more bugs are fixed, more features are added, and I see the semblance of stability.

Personally, I will always recommend Python + Django or Ruby on Rails as a backend before NodeJS.  Those frameworks and their ORMs still outshine whatever is available in the JavaScript ecosystem.  The only exceptions would be if your main focus (and the team’s main focus) is JavaScript or if you are working heavily in serverless applications.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
